commands/bootstrap/index.js

```diff
@@ -104,7 +106,7 @@ class BootstrapCommand extends Command {
 
     // root install does not need progress bar
     this.enableProgressBar();
-    this.logger.info("", `Bootstrapping ${this.filteredPackages.length} packages`);
```

Comment:
The `Bootstrapping` (begin) and `Bootstrapped` (end) labels have been reversed, needs to be restored.

Reply:
And the snapshots need to be updated for the bootstrap integration test. `npm run integration -- -u`

Reply:
@evocateur made the changes.